lib/presentation/map_tokyo/map_tokyo_page.dart

```diff
@@ -0,0 +1,140 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:location_app/presentation/map_tokyo/map_tokyo_model.dart';
+import 'package:location_app/presentation/speed_dial/my_speed_dial.dart';
+import 'package:percent_indicator/circular_percent_indicator.dart';
+import 'package:provider/provider.dart';
+
+class MapTokyoPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return ChangeNotifierProvider<MapTokyoModel>(
+      create: (_) => MapTokyoModel()..getMapTokyoInformationRealTime(),
+      child: Scaffold(
+        body: Consumer<MapTokyoModel>(
+          builder: (context, model, child) {
+            final imageTokyoWardList = model.mapTokyoWardInformationList;
+            final mapButtonInformationList =
+                model.mapTokyoWardButtonInformationList;
+            final mapInfo = imageTokyoWardList
+                .map((mapTokyoWardInformation) => Image.asset(
+                    "images/tokyo/${mapTokyoWardInformation.documentId}.png"))
+                .toList();
+            final mapButtons = mapButtonInformationList
+                .map((mapTokyoWardButtonInformation) => FlatButton(
+                      color: mapTokyoWardButtonInformation.disFlag
+                          ? Theme.of(context).primaryColor
+                          : Colors.grey,
+                      child: Text(
+                        mapTokyoWardButtonInformation.tokyoWardName,
+                        style: TextStyle(
+                          fontWeight: FontWeight.bold,
+                          fontSize: 9,
```

Comment:
```suggestion
                                                  fontSize:
                            mapTokyoButtonInformation.tokyoWardName.length == 2
                                ? 15
                                : 14,
,
```

ここもmapPageと同じでいいような気がします！！
あと、これはfirebase上の修正となりますが、「区」は無しでもいいかもですね！

Reply:
b077100a722895078b37926b21945cc889e0e1d4
にてボタンの修正等を行いました！
確認よろしくお願いします。

Reply:
> あと、これはfirebase上の修正となりますが、「区」は無しでもいいかもですね！
> あと、文京区のtokyoWardNameが"荒川区"になってますので、お手数ですが修正お願いします！

上記に対する修正確認しました！ありがとうございます！